packages/video_player/video_player_avfoundation/example/ios/RunnerUITests/VideoPlayerUITests.m

```diff
@@ -30,16 +30,35 @@ - (void)testPlayVideo {
   XCTAssertTrue([playButton waitForExistenceWithTimeout:30.0]);
   [playButton tap];
 
-  XCUIElement *playbackSpeed1x = app.staticTexts[@"Playback speed\n1.0x"];
-  XCTAssertTrue([playbackSpeed1x waitForExistenceWithTimeout:30.0]);
+  XCUIElement *playbackSpeed1x = app.staticTexts[@"1.0x\nPlayback speed"];
+  BOOL foundPlaybackSpeed1x = [playbackSpeed1x waitForExistenceWithTimeout:30.0];
+  if (!foundPlaybackSpeed1x) {
+    // Some old flutter version uses a different accessibility value for tooltips.
+    // We try to find the button if the old accessibility value is used.
+    // TODO(cyanglaz): Remove this when the new accessibility value is supported on flutter/stable
+    // https://github.com/flutter/flutter/issues/102771
+    playbackSpeed1x = app.staticTexts[@"Playback speed\n1.0x\n"];
+    foundPlaybackSpeed1x = [playbackSpeed1x waitForExistenceWithTimeout:30.0];
+  }
+  XCTAssertTrue(foundPlaybackSpeed1x);
   [playbackSpeed1x tap];
 
   XCUIElement *playbackSpeed5xButton = app.buttons[@"5.0x"];
   XCTAssertTrue([playbackSpeed5xButton waitForExistenceWithTimeout:30.0]);
   [playbackSpeed5xButton tap];
 
-  XCUIElement *playbackSpeed5x = app.staticTexts[@"Playback speed\n5.0x"];
-  XCTAssertTrue([playbackSpeed5x waitForExistenceWithTimeout:30.0]);
+  XCUIElement *playbackSpeed5x = app.staticTexts[@"5.0x\nPlayback speed"];
+  BOOL foundPlaybackSpeed5x = [playbackSpeed5x waitForExistenceWithTimeout:30.0];
```

Comment:
These two checks will add a full minute.  You can make a predicate that waits for either, or you can shorten this wait considerably (like 2 seconds).

Reply:
Done, i shorten the second wait. I like this current code ordering because it is easier to remove later.

Reply:
I think it makes sense to shorten the first one and not the second. since that's the one that may fail if it's on a version that only handles the second one.

Reply:
I think it is necessary to wait for the first one to handle things like lags on the test environment. We don't need much waiting for the 2nd because we know after 30 seconds of waiting, it is almost guaranteed that the page is stable.